MTFH/Accepted/Integration-with-mmh.md

```diff
@@ -0,0 +1,81 @@
+#  Integration with MMH-derived services (Assets API / Housing Search API)
+
+### **Date:** 15th September 2021
+
+### **Status:** ACCEPTED
+
+## **Context**
+
+The repairs context has evolved naturally in the context of Modern Tools for Housing, it's parent programme, as a separate discrete service. Since the cyber attack, this has been developed in isolation from the other MTFH services, sharing only a data store. The time has come to integrate Repairs with the services previously developed by Manage my Home (formerly Tenants & Leaseholders). We will be starting with two integration points:
+
+1) The Housing Search API during the property search flow
+2) The Assets API during the raise a repair flow
+
+Two options for integration were discussed:
+
+## Option 1: Integrate at the API level with Assets API and Housing Search API
+
+Property search flow:
+1. User enters search screen and enters prop address
+2. API call to property search API
```

Comment:
I am not sure if we made a decision on it, but would it be possible if we call the Housing Search API and Assets API directly from the FE, instead of from the Repairs service API?

Reply:
Yes, thanks, I will clarify